CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+### Added
+* Added support for XCFramework. AmazonChimeSDK and AmazonChimeSDKMedia binaries now contain .xcframework format as well. See `README.md` for updated Setup instructions.
```

Comment:
Is this only document change? Should we add [Document] prefix?

Reply:
This is not just documentation though, there are some build changes and XCFrameworks are in the binaries now that builders can choose between .framework and .xcframework.